src/sezzle-checkout-button/sezzle-checkout-button-asset.js

```diff
@@ -97,3 +97,41 @@ class SezzleCheckoutButton {
 		this.createButton()
 	}
 }
+
+class EventLogger {
+	constructor(options){
+		this.merchantUUID = options.merchantUUID || '';
+	}
+
+	sendEvent(eventName, description="") {
+		const url = "http://widget.sezzle.com/v1/event/log";
```

Comment:
I am confused about the duality of this whole checkout button work. Are we making it for the users manual install or for the shopify app UI? 

In the first case, we'd better to have this URL hardcoded to simplify the code that users have to embed manually (look the readme)

In case of shopify app UI, we'd rather move the URL to the init parameter, so that the SG can choose which server to send the event on different environments. Otherwise, we'll end up with the Staging app sending events to the Production. 

Probably need some feedback from @aholdahl  here

Reply:
When we add this script via SG/SFS I planned on checking the env and replacing the domain accordingly. Let me know your thoughts.

Reply:
Will you have to scan the file for the prod domain name and replace it accordingly?

Reply:
Right, and since it would only run for local and test there won't be any impact to prod latency.

Reply:
I guess this is done to simplify the snippet code? 
Ok, to simplify the snippet code, what we can do is:
1. instantiate the `EventLogger` in the `SezzleCheckoutButton` constructor
2. wrap the `this.createButton()` line in the `init` function in `try..catch` with logging `onload` and `error` correspondingly (similar to what is done in the snippet code)
3. in the `SezzleCheckoutButton` constructor, default the URL to the prod if not provided
4. for the manual installation, the merchant should provide only `merchant_UUID` if no theme customization required. So readme has just the:
`
new SezzleCheckoutButton({ merchant_uuid: "smth"}).init()
`
5. for the UI installation, SFS provides the URL in the snippet.

Let me know what you think

Reply:
addressed